### PR TITLE
[Merged by Bors] - bevy_render: add a small color box to each color

### PR DESCRIPTION
**Pull request by light4**
# Objective

- bevy_render: add a small color box to each color

## Solution

bevy_color_doc

---

### CODE REVIEW

**Review by DJMcNab — APPROVED** [review]
This is a great idea, and it looks like it turns out quite nicely.

File: crates/bevy_render/src/color/mod.rs [hunk 1]
```diff
@@ -47,43 +47,81 @@ pub enum Color {
 }
 
 impl Color {
+    /// <div style="background-color:rgb(240, 248, 255); width: 10px; padding: 10px; border: 1px solid;"></div>
     pub const ALICE_BLUE: Color = Color::rgb(0.94, 0.97, 1.0);
```
Comment (mockersf) [inline crates/bevy_render/src/color/mod.rs:51]:
```suggestion
    /// <div style="background-color:rgb(94%, 97%, 100%); width: 10px; padding: 10px; border: 1px solid;"></div>
    pub const ALICE_BLUE: Color = Color::rgb(0.94, 0.97, 1.0);
```
Can you put the colors with percentages? It's easier to have the same values that way

Comment (light4) [reply crates/bevy_render/src/color/mod.rs:51]: ok

**Review by cart — APPROVED** [review]
Clever. I like it!

**Comment by alice-i-cecile** [comment]
I'm a bit late, but I love this @light4. Thanks!

**Pull request closed without merging**